src/Sylius/Component/Core/Uploader/PathGeneratorInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Sylius\Component\Core\Uploader;
+
+use Gaufrette\Filesystem;
```

Comment:
Good point, in fact the most implementations will use only one method of it. :+1: